Assets/GameLogic/StoreLogicService/StoreLogicService.cs

```diff
@@ -3,19 +3,63 @@
 using System.Linq;
 using System.Text;
 
-namespace ThirdEyeSoftware.GameLogic
+namespace ThirdEyeSoftware.GameLogic.StoreLogicService
 {
     public interface IStoreLogicService
     {
         IDataLayer DataLayer { get; set; }
         void OnAppStorePurchaseSucceeded(string productId);
         Action<string> LogToDebugOutput { get; set; }
+        List<string> ValidateProducts(List<ProductInfo> products);
     }
 
     public class StoreLogicService : IStoreLogicService
     {
         private static readonly StoreLogicService _instance = new StoreLogicService();
 
+        private ProductInfo FindSmallestProductInfo(List<ProductInfo> productInfos)
+        {
+            foreach (var element in productInfos)
+            {
```

Comment:
Can you please rename `element` to `curProductInfo`? This is the naming convention for all foreach iteration vars in the whole project.